src/components/Postiterie.vue

```diff
@@ -1,203 +1,454 @@
 <template>
-  <div>
-    <div class="control">
-      <label  for="file" class="label-input"><span class="button">Choisir une image</span></label>
-      <input id="file" type="file" class="button-input" @change="previewImage($event.target.name, $event.target.files)" accept="image/png, image/jpeg">
-      <span id="startTracking"  class="button" v-bind:class="{faded: isNoImage}" @click="tracking">Traiter</span><br>
+
+  <div class="container">
+    <div class="button-container">
+      <div class=" button-top button-orange"><i class="material-icons">photo_camera</i></div>
+      <span class="button button-orange">Prendre une photo</span>
     </div>
-    <div class="frame">
-      <div class="container">
-        <img id="preview" :src="image" alt="">
-      </div>
+
+    <div class="small-button-container">
```

Comment:
You can probably put every small UI part (buttons, fields, headers ...) into their own vue components